SPECS/bison/bison.spec

```diff
@@ -42,12 +46,16 @@ make %{?_smp_mflags} check
 %{_datarootdir}/aclocal/*
 %{_mandir}/*/*
 %{_docdir}/bison/*
+
 %changelog
-* Sat May 09 00:21:04 PST 2020 Nick Samson <nisamson@microsoft.com> - 3.1-3
+* Fri Aug 21 2020 Thomas Crain <thcrain@microsoft.com> 3.1-4
+- Remove yacc command for compatibility with byacc package
+- Remove sha hash
+- License verified
+* Sat May 09 00:21:04 PST 2020 Nick Samson <nisamson@microsoft.com> 3.1-3
 - Added %%license line automatically
-
-*   Tue Sep 03 2019 Mateusz Malisz <mamalisz@microsoft.com> 3.1-2
--   Initial CBL-Mariner import from Photon (license: Apache2).
```

Comment:
Please revert to the change to this entry - it was correct and the new one is the outdated version.

Please also double-check this hasn't happened in other modified spec files.

Reply:
Done, reverted the change in python-docutils.spec also